Plugins/NativeServerPlugin/inc/conductor.h

```diff
@@ -142,26 +139,31 @@ class Conductor : public webrtc::PeerConnectionObserver,
 
 	void OnFailure(const std::string& error) override;
 
+private:
+	void SendMessageToPeer(void* data);
```

Comment:
these being `void*` is an immediate red flag for me... 😅 i'm going to keep reading and try to understand more before i can suggest an alternative though.

Reply:
okay, so looking below it does seem like these can only be a single data type, so can we change these to the explicit type? for example: `webrtc::MediaStreamInterface*`? it will improve readability and understanding via intelisense/similar

Reply:
I totally agree with you. This is another issue of the code refactoring which I did to conductor class.